src/session9/challenges/college_management_system/Professor.java

```diff
@@ -0,0 +1,81 @@
+package session9.challenges.college_management_system;
+
+public class Professor {
+
+    private String firstName;
+    private String lastName;
+    private String sex;
+    private int age;
+    private String dateOfBirth;
```

Comment:
Please consider using a class that we can use to manipulate dates (LodalDateTime maybe?).